### PR TITLE
Static Dashboard UI

### PR DESCRIPTION
**Pull request by matzapata**
## Overview

Jira tickets: [SAF-28](https://wakeuplabs.atlassian.net/browse/SAF-28)  [SAF-27](https://wakeuplabs.atlassian.net/browse/SAF-27)  [SAF-31](https://wakeuplabs.atlassian.net/browse/SAF-31)

Dashboard ui static implementation. Logic implementation and maybe some UI fixes/improvement will follow up as small prs. Routing portion is mostly temporary, we can adjust that later when fully ready to integrate.

## Notes

Components from -> https://dev--sovryn-storybook.netlify.app/

Code style from -> https://build.sovryn.com/builder-portal/ui-library/working-with-components 

Particularly referenced the LendPage for code style -> https://github.com/DistributedCollective/sovryn-dapp/tree/develop/apps/frontend/src/app/5_pages/LendPage

Had to make some minimal changes in components library to match design styles and fix some bugs, it should not have any impact in the rest of the codebase, but I can try to find a workaround if necesary.

## Demo

https://github.com/user-attachments/assets/59ca13a2-6a7f-4678-bf18-66ca5d4394bb

## Designs

https://www.figma.com/design/T97rmXXdXJHkrrjaIVhHPu/D2---Borrow%2FLend-Aave?node-id=33-43633&t=bRhmevLKUOzmpwqs-0

### CODE REVIEW

File: apps/frontend/src/app/5_pages/AavePage/AaavePage.tsx [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+          onChange={e => setActiveLiquidityTab(e)}
+          size={TabSize.normal}
+          type={TabType.secondary}
+        />
+
+        {/* Lending and borrowing columns */}
+        <div className="grid grid-cols-1 lg:grid-cols-2 lg:gap-5">
+          <div
+            className={classNames(
+              activeLiquidityTab !== LiquidityTabs.LEND ? 'hidden' : '',
```
Comment (njaume) [inline apps/frontend/src/app/5_pages/AavePage/AaavePage.tsx]: Here's an example of how you can conditionally apply classes using classnames.

const myComponentClasses = classNames({
    'myClass': true,
    'is-active': condition,
    'has-icon': anotherCondition
  });

File: apps/frontend/src/app/5_pages/AavePage/AaavePage.tsx [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+            )}
+          >
+            <LendPositionsList />
+            <LendAssetsList />
+          </div>
+
+          {/* Borrowing column */}
+          <div
+            className={classNames(
+              activeLiquidityTab !== LiquidityTabs.BORROW && 'hidden',
```
Comment (njaume) [inline apps/frontend/src/app/5_pages/AavePage/AaavePage.tsx]: Same!

**Review by njaume — APPROVED** [review]
LGTM! just a few comments no changes needed

**Review by juandahl — APPROVED** [review]
LGTM. Nice code 👍

**Pull request merged**